code/modules/recycling/conveyor2.dm

```diff
@@ -50,6 +50,10 @@
 		dir = newdir
 	update_move_direction()
 
+/obj/machinery/conveyor/set_dir(newdir)

```

Comment:
We should probably do what TG does with this, some time (where all directionalities are replaced with `setDir`.